arc/src/CuriePME.c

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) 2017, Intel Corporation.
+
+#include "CuriePME.h"
```

Comment:
Minor but let's not start capitalizing c source files... zjs_arc_curie_pme maybe?

Or zjs_curie_pme or maybe just curie_pme, I don't know anymore. :)